accio:
  hardware_interface:
    loop_hz: 10 # hertz Hz
    joints:
      - FL_hinge
      - FR_hinge
      - RL_hinge
      - RR_hinge